lib/utils/getCommandSuggestion.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const chai = require('chai');
+const getCommandSuggestion = require('./getCommandSuggestion');
+chai.use(require('chai-as-promised'));
+const expect = require('chai').expect;
+const Serverless = require('../../lib/Serverless');
+
+const serverless = new Serverless();
+serverless.init();
+
+describe('#getCommandSuggestion', () => {
+  it('should return "package" as a suggested command if you input "pekage"', () =>
+    expect(getCommandSuggestion('pekage', serverless.cli.loadedCommands))
```

Comment:
getCommandSuggection() returns a promise, so you must return the expectation here:
```
return expect(....)
```